Formula/pfetch-rs.rb

```diff
@@ -0,0 +1,18 @@
+class PfetchRs < Formula
+  desc "Pretty system information tool written in Rust"
+  homepage "https://github.com/gobidev/pfetch-rs"
+  url "https://github.com/gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
```

Comment:
You don't need to change these, actually. The audit was complaining about the `HEAD`.

```
HEAD: The source URL https://github.com/Gobidev/pfetch-rs is not reachable
```

```suggestion
  homepage "https://github.com/Gobidev/pfetch-rs"
  url "https://github.com/Gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
```

Reply:
Yes I saw your comment after tge fact.